internal/region/region.go

```diff
@@ -10,6 +10,8 @@ const (
 	USWest1
 	// EUCentral1 is a sauce labs region in the EU, aka eu-central-1.
 	EUCentral1
+	// APACSoutheast is a sauce labs region in the APAC, aka apac-southeast-1
+	APACSoutheast
```

Comment:
Keeping consistency (see `USWest1` and `EUCentral1`)
```suggestion
	APACSoutheast1
```